server/budgettracker.data/Converters/BudgetConverter.cs

```diff
@@ -8,27 +8,35 @@
 
 namespace budgettracker.data.Converters
 {
-    public class BudgetConverter : IConverter<Budget, BudgetModel>
+    public class BudgetConverter
     {
-        public Budget ToBusinessModel(BudgetModel dataModel)
+        public static Budget ToBusinessModel(BudgetModel dataModel)
         {
-            return new Budget()
+            Budget budget = new Budget()
             {
                 Id = dataModel.Id,
                 Name = dataModel.Name,
                 SetAmount = dataModel.SetAmount,
                 Duration = GetBudgetDuration(dataModel.Duration),
                 BudgetStart = dataModel.BudgetStart,
-                ParentBudgetId = dataModel.ParentBudgetId
+                ParentBudgetId = dataModel.ParentBudgetId,
+                Owner = GetOwnerForBudget(dataModel),
```

Comment:
I think we should reconsider how we use these converters kinda holding a lot of logic that should be a simple assignment.

Reply:
It can't be a simple assignment because it's also converting from one class to another.

Reply:
Maybe we could have a BudgetConverterHelper file that has static methods to handle this.

Reply:
This isn't a helper though. It is a converter. I also don't like "helper" classes cause all that means is that we don't know where else to put it.

Reply:
Fair enough, converter is fine.